api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -541,18 +588,17 @@ private void sendWithRetries(
       String from,
       List<String> toRecipientEmails,
       List<String> ccRecipientEmails,
+      List<String> bccRecipientEmails,
```

Comment:
any particular reason to add BCCs at this time? clean way to do so, so I'm fine with it

Reply:
Added to the MailService method documentation - PTAL there

Reply:
oops, I was confused.  I didn't see that this was actually **used**.  never mind.